tasks/db_deploy/test/unit_tests/test_migrate_db.py

```diff
@@ -18,43 +18,34 @@ def setUp(self):
         """
         Set up test.
         """
-        self.config = {
-            "host": "aws.postgresrds.host",
-            "port": "5432",
-            "database": "disaster_recovery",
-            "admin_database": "postgres",
-            "app_user": "orcauser",
-            "admin_user": "postgres",
-            "app_user_password": "MySecretUserPassword",
-            "admin_user_password": "MySecretAdminPassword",
-        }
+        self.config = {"admin_database":"admin_db", "admin_password":"admin123", "admin_username":"admin", "host":"aws.postgresrds.host", "port":5432, "user_database":"user_db", "user_password":"user123", "user_username":"user"}
 
     def tearDown(self):
         """
         Tear down test
         """
         self.config = None
 
-    @patch("migrate_db.migrate_versions_1_to_2")
```

Comment:
Don't we still need to test 1 to 2?

Reply:
depending upon if we want to keep v1 code.

Reply:
I concur, you should keep version 1 - 2 migration code and tests. @AndrewDorn  and @rizbi-hassan  would it make sense to break the specific migrations, i.e 1 - 2 and 2 - 3 into their own python files and test files? For the sake of easier management and maintenance?